src/app/form-styles/form-list/form-list.component.html

```diff
@@ -0,0 +1,76 @@
+<div class="stylesContainer">
+   <ng-container *ngFor="let item of (listStylesForm$|async) | keyvalue">
```

Comment:
use ngrxPush instead of async

How PushPipe and the let-directive improve performance?
PushPipe and the let-directive improve performance in two ways:

Only trigger change detection when a new observable value is emitted
Trigger change detection only for the component and its children

https://christiankohler.net/reactive-angular-with-ngrx-component

```suggestion
   <ng-container *ngFor="let item of (listStylesForm$ | ngrxPush) | keyvalue">
```